.pylintrc

```diff
@@ -85,6 +93,7 @@ ignore-imports=yes
 [MESSAGES CONTROL]
 
 disable=abstract-method,
+        ungrouped-imports,
```

Comment:
Are we expecting to add new items to this list? (considering we are working on enabling this check through a different github-issue.)

Reply:
I added this because the grouping is already handled by the sort library and it just makes things confusing in some cases.